core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.ts

```diff
@@ -101,7 +101,10 @@ export class DotEditContentCategoryFieldComponent implements OnInit {
         effect(
             () => {
                 const categoryValues = this.store.selectedCategoriesValues();
-                this.categoryFieldControl.setValue(categoryValues);
+
+                if (this.categoryFieldControl) {
```

Comment:
@oidacra this `if` is necessary? The idea of declaring the effect in the `ngOnInit` method is to avoid this `if` because at that point `this.categoryFieldControl` should exist or not?

Reply:
Take a look later.

Reply:
@oidacra If needed, you can use `this.categoryFieldControl?.setValue(categoryValues)`  instead of the conditional